cime_config/machines/Depends.crusher-gpu.crayclang.cmake

```diff
@@ -0,0 +1,43 @@
+set(CICE_F90
```

Comment:
i think e3sm won't use crusher-gpu machine (it is either crusher+crayclang or crusher+crayclanggpu), so this file is not needed?

related -- there are 2 files in cmake_macros, crayclang_crusher-cpu and crayclang_crusher-gpu, they are also not needed.

Reply:
i see that in config_machines, crusher-cpu and crusher-gpu blocks made in. they need to be removed. in scream, we are trying to adopt e3sm machines' names instead (so, crusher+crayclang and crusher+crayclanggpu).

Reply:
Done.

Reply:
I think based on comments below from @ambrad we should revert the commits that removed these configs.